docker-compose.yml

```diff
@@ -3,7 +3,7 @@ services:
     build:
       context: .
       dockerfile: Dockerfile
-    image: betterstack/collector:latest
+    # image: betterstack/collector:latest
```

Comment:
@paweljw was this intentional, please?

Reply:
😱 no, https://github.com/BetterStackHQ/collector/pull/19